demo/primefaces-users-portlet/src/main/webapp/WEB-INF/liferay-plugin-package.properties

```diff
@@ -8,4 +8,6 @@ page-url=http://www.liferay.com/community/liferay-projects/liferay-faces
 author=liferay.com
 licenses=LGPL 2.1
 liferay-versions=7.0.*
+Bundle-SymbolicName: com.liferay.faces.portal.demo.primefaces.users
```

Comment:
@stiemannkj1 I removed it in the code review and it works. The WAB generator creates a MANIFEST.MF with:
```
Bundle-SymbolicName: com.liferay.faces.demo.primefaces.users.portlet
```